OrbitGl/ThreadTrack.cpp

```diff
@@ -126,8 +129,9 @@ void ThreadTrack::SetTimesliceText(const TimerInfo& timer_info,
   TimeGraphLayout layout = time_graph_->GetLayout();
   if (text_box->GetText().empty()) {
     std::string time = GetPrettyTime(absl::Microseconds(elapsed_us));
-    FunctionInfo* func =
-        Capture::GSelectedFunctionsMap[timer_info.function_address()];
+    const FunctionInfo* func = Capture::capture_data_.GetSelectedFunction(
+        timer_info.function_address());
+    CHECK(func != nullptr);
```

Comment:
Two lines after this CHECK there is `if(func)`.

Reply:
Thanks. I removed the check here, as it seems that the introspected functions can also occur here.